appveyor.yml

```diff
@@ -24,6 +24,8 @@ init:
 
 install:
   - set PATH=C:\Ruby26-x64\bin;%PATH%
+  - gem update --system
+  - FOR /F "usebackq delims==" %%A IN (`ruby appveyor/bundler_version.rb`) DO gem install bundler -v %%A
```

Comment:
this to ensure we use the version we bundled in the Gemfile.lock. Matches what we do on other CI environments